models/lobby.go

```diff
@@ -260,6 +260,19 @@ func (lobby *Lobby) UnreadyPlayer(player *Player) *helpers.TPError {
 	return nil
 }
 
+func (lobby *Lobby) RemoveUnreadyPlayers() {
+	var slots []*LobbySlot
+	db.DB.Where("lobby_id = ?", lobby.ID).Find(slots)
+
+	for _, slot := range slots {
+		if !slot.Ready {
+			db.DB.Where("player_id = ? AND lobby_id = ?", slot.ID, lobby.ID).Delete(&LobbySlot{})
```

Comment:
You don't need to actually retrieve the slots. You can just do `db.DB.Where("player_id = ? AND lobby_id = ? AND ready = false", slot.ID, lobby.ID).Delete(&LobbySlot{})`. Also, this function and UnreadyAllPlayers should have tests.